packages/forma-36-react-components/src/components/Table/Table.css

```diff
@@ -1,6 +1,13 @@
 @import 'settings/colors';
+@import 'settings/borders';
 
-.Table {
+.Table,
+.TableContainer {
   width: 100%;
+}
+
+.TableContainer--inline {
   border: 1px solid var(--color-element-light);
+  border-radius: var(--border-radius-medium);
+  overflow: hidden;
```

Comment:
Needed for the border clipping? Hopefully this doesn't cause any issues

Reply:
Yes it seems to be necessary for the border-radius to take effect in all browsers.

Reply:
I've pushed an alternative.